code/modules/mob/mob.dm

```diff
@@ -1495,11 +1495,6 @@ GLOBAL_LIST_INIT(holy_areas, typecacheof(list(
 	if(!mind)
 		return FALSE
 
-	//Allows fullpower vampires to bypass holy areas
-	var/datum/vampire/vampire = mind.vampire
-	if(vampire && vampire.get_ability(/datum/vampire_passive/full))
-		return FALSE
-
```

Comment:
Is there a reason this is being removed?

Reply:
It's not used. Vampires have their own seperate check. This proc isnt used by any vampire related stuff.